src/LemonBot.Web/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -4,6 +4,6 @@ namespace LemonBot.Web.Extensions;
 
 public static class ClaimsPrincipalExtensions
 {
-    public static string GetUserId(this ClaimsPrincipal user)
-        => user.FindFirstValue(ClaimTypes.NameIdentifier);
+	public static string? GetUserId(this ClaimsPrincipal user)
+		=> user.FindFirstValue(ClaimTypes.NameIdentifier);
```

Comment:
This call cannot return null. I will add a ! to ensure the value is not null (or return a string.Empty)